src/Standards/PSR12/Sniffs/ControlStructures/BooleanOperatorPlacementSniff.php

```diff
@@ -66,8 +74,13 @@ public function process(File $phpcsFile, $stackPtr)
             T_BOOLEAN_OR,
         ];
 
+        if ('first' === $this->allowOnly || 'last' === $this->allowOnly) {
```

Comment:
Caught by the new Yoda conditions check :) I only just added this in to ensure consistency with the rest of the codebase.

Reply:
Damn. I get so much error with this project because the rules are really different than the one I use. This should be fixed now ! :)